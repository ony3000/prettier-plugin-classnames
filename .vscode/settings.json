{
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "eslint.workingDirectories": [{ "mode": "auto" }]
}
