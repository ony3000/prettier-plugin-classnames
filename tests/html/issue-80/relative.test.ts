import { thisPlugin, testSnapshotEach } from '../../adaptor';
import { baseOptions } from '../../settings';
import { fixtures } from './fixtures';

const options = {
  ...baseOptions,
  plugins: [thisPlugin],
  parser: 'html',
  endingPosition: 'relative',
};

testSnapshotEach(fixtures, options);
