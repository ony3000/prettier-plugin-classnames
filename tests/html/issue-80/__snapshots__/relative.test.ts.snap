// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'(1) If a script tag has no type, it i…' > expectation 1`] = `
"<script>
  export function Foo({ children }) {
    return <div className="lorem ipsum dolor sit amet">{children}</div>;
  }
</script>
"
`;

exports[`'(2) If a script tag has an empty type…' > expectation 1`] = `
"<script type="">
  export function Foo({ children }) {
    return <div className="lorem ipsum dolor sit amet">{children}</div>;
  }
</script>
"
`;

exports[`'(3) If a script tag has a type of \`te…' > expectation 1`] = `
"<script type="text/javascript">
  export function Foo({ children }) {
    return <div className="lorem ipsum dolor sit amet">{children}</div>;
  }
</script>
"
`;

exports[`'(4) If a script tag has a \`lang\` attr…' > expectation 1`] = `
"<script lang="ts" type="anything">
  interface MyInterface {
    foo(): string;
    bar: Array<number>;
  }
</script>
"
`;

exports[`'(5) Otherwise, leave Prettier\\'s outp…' > expectation 1`] = `
"<script type="application/json">
  {
    "lorem": "ipsum",
    "dolor": "sit amet"
  }
</script>
"
`;
